static/css/index.css

```diff
@@ -402,3 +410,114 @@ header {
 }
 
 /* end content styles */
+
+/* start media query styles */
+@media screen and (min-width: 700px) {
+  .container {
+    margin: 0 auto;
+    width: 90%;
+  }
+
+  .header__container__parhead__nav__ul > .iwq {
+    display: flex;
+  }
+
+  .header__container__parhead__nav__ul__responsive {
+    width: 0;
+    height: 0;
+  }
+
+  .sectionone {
+    height: 916px;
+  }
+
+  .overlay {
+    background-image: url(../imgs/Header\ 2x.png);
+    height: 916px;
+    background-size: contain;
+  }
+
+  .sectiontwo__container__items,
+  .sectionthree__container__posts,
+  .sectionfour__container__rows,
+  .sectionfive__container__services {
+    flex-direction: row;
+    display: grid;
+    grid-column-gap: 30px;
+    border: 2px solid #ddd;
+    border-radius: 19px;
+    background-color: #fff;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 452px;
+    padding: 21px;
+  }
+
+  .sectionsix__container__skills {
+    flex-direction: row;
+    display: grid;
+    grid-column-gap: 39px;
+    padding: 15px 15px 33px 15px;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 481px;
+  }
```

Comment:
- [ ] As the last reviewer said 👀: ** The right-hand side should be taking more space than the left. About 40% for left and 60% for right**

| **Figma template** |  **Your design** |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/193029570-8d4f77ec-b693-4bae-8222-9bc9e469d01f.png) | ![image](https://user-images.githubusercontent.com/58642949/193029721-cd03931b-ac72-4239-bc26-c9047ca7a90f.png) |